File: api/src/models/ContactosModel.js
/*
    Esquema de para la colección Contactos en la base de datos
    En los esquemas deben poner todos los atributos de la base de datos para poder manipularlos.
*/

const mongoose = require('mongoose');
const ContactosSchema = new mongoose.Schema({

    _id: { type: Number},
    telefono: { type: String},
    correo_electronico: { type: String},
    redes_sociales: 
        {
            facebook: { type: String},
            twitter: { type: String},
            instagram: { type: String},
        }
    
},{ collection: 'Contactos' });

module.exports = mongoose.model('Contactos', ContactosSchema)

File: web/MongoProyectoWeb/MongoProyectoWeb/Models/ContactosModel.cs
using MongoProyectoWeb.Models;

namespace MongoProyectoWeb.Models
{
    public class ContactosModel
    {
        public int? _id { get; set; }
        public string telefono { get; set; }
        public string correo_electronico { get; set; }
        public RedesSocialesModel redes_sociales { get; set; } = new RedesSocialesModel();
    }

    public class RedesSocialesModel
    {
        public string facebook { get; set; }
        public string twitter { get; set; }
        public string instagram { get; set; }
    }


}